### ISSUE TITLE
Popover tabindex when reference is an input

### ISSUE THREAD

**Issue by aphofstede** [ElemeFE/element#11512]
### Element UI version
2.4.0

### OS/Browsers version
Chrome 66 macOS

### Vue version
2.5.16

### Reproduction Link
[https://jsfiddle.net/mmx38qxw/2021/](https://jsfiddle.net/mmx38qxw/2021/)

### Steps to reproduce
Create a Popover with an input as a "reference" element. The form field now needs two tab presses to get focus. (Caused by the tabindex being set to '0' hard-coded for the reference field)

Work-around:
```
    <popover ref="elPopover" ...>
    ...
    mounted() {
        this.$refs.elPopover.referenceElm.setAttribute('tabindex', -1)
    },
```

Related: https://github.com/ElemeFE/element/issues/9038

### What is Expected?
The native reference element (input, textarea, etc.) gets the tabindex

### What is actually happening?
The reference element's container gets a tabindex

**Comment by atilkan** [comment]
Yes, it is hardcoded. This should be optional. 

![screen shot 2018-06-09 at 01 03 16](https://user-images.githubusercontent.com/1254613/41182898-6f4a93e6-6b80-11e8-85df-098aa8a4e773.png)

**Comment by ziyoung** [comment]
ref #11875

**Comment by fe-leon** [comment]
i can't stand this
